lib/tasks/fix_fdw_credentials.rake

```diff
@@ -6,31 +22,33 @@ end
 
 def fix_primary_server_fdw_credentials
   config = ActiveRecord::Base.configurations['secondbase'][Rails.env]
+  username = ActiveRecord::Base.configurations[Rails.env]['username']
```

Comment:
Have the .env files with this new configuration already been pushed to master?

Reply:
Yeah, they are in the openhub-config directory and are updated.